proposals/compatibility-mode.md

```diff
@@ -14,33 +14,86 @@ The primary goal of WebGPU Compatibility mode is to increase the reach of WebGPU
 
 Since WebGPU Compatibility mode is a subset of WebGPU, all valid Compatibility mode applications are also valid WebGPU applications. Consequently, Compatibility mode applications will also run on user agents which do not support Compatibility mode. Such user agents will simply ignore the option requesting a Compatibility mode Adapter and return a Core WebGPU Adapter instead.
 
-## WebGPU Spec Changes
+# WebGPU Spec Changes
 
-When calling `GPU.requestAdapter()`, passing `featureLevel = "compatibility"` in the `GPURequestAdapterOptions` will indicate to the User Agent to select the Compatibility subset of WebGPU. Any Devices created from the resulting Adapter on supporting UAs will support only Compatibility mode. Calls to APIs unsupported by Compatibility mode will result in validation errors.
+Spec changes are described in the following subsections.
 
-Note that a supporting User Agent may return a `featureLevel = "compatibility"` Adapter which is backed by a fully WebGPU-capable hardware adapter, such as D3D12, Metal or Vulkan, so long as it validates all subsequent API calls made on the Adapter and the objects it vends against the Compatibility subset.
+## Validation
 
-```webidl
-partial interface GPUAdapter {
-    readonly attribute DOMstring featureLevel;
-}
-```
+As always, API calls which are unsupported by a Device will result in validation errors.
+Even if a Device is backed by a fully WebGPU-capable hardware adapter, such as D3D12, Metal or Vulkan, it validates all subsequent API calls made on the Adapter and the objects it vends according the set of features enabled on the device, including Compatibility mode validation when `"webgpu-core"` is not enabled.
 
-As a convenience to the developer, the Adapter returned will have the `featureLevel` property set to `"compatibility"`.
+Note there are currently no "Compatibility features", features that would provide incremental capabilities _between_ Compatibility and Core modes.
+See [discussion](https://github.com/gpuweb/gpuweb/issues/4987#issuecomment-2625791411).
 
-```webidl
-partial dictionary GPUTextureDescriptor {
-    GPUTextureViewDimension textureBindingViewDimension;
+## `"webgpu-core"` Feature
+
+A new feature `"webgpu-core"`, when enabled on a device, lifts all Compatibility mode restrictions (features and limits). See below for details.
+
+TODO: Bikeshed the name and replace all instances.
+
+- `"webgpu-core"` (kinda redundant)
+- `"core"` (possibly slightly confusing with `featureLevel: "core"`)
+- `"core-features-and-limits"` (clear but verbose)
```

Comment:
bikeshedding

`"feature-level-core"` or `"featurelevel-core"`

Reply:
those aren't bad, somewhat clearer relationship to featureLevel